gen/azure/cloud-config.yaml

```diff
@@ -11,7 +11,7 @@ root:
       StandardOutput=journal+console
       StandardError=journal+console
       ExecStartPre=/usr/bin/curl -fLsSv --retry 20 -Y 100000 -y 60 -o /var/tmp/d.deb https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_17.09.0~ce-0~ubuntu_amd64.deb
-      ExecStart=/usr/bin/bash -c "try=1;until dpkg -D3 -i /var/tmp/d.deb || ((try>9));do echo retry $((try++));sleep $((try*try));done;systemctl --now start docker;systemctl restart docker.socket"
+      ExecStart=/usr/bin/env bash -c "try=1;until dpkg -D3 -i /var/tmp/d.deb || ((try>9));do echo retry $((try++));sleep $((try*try));done;systemctl --now start docker;systemctl restart docker.socket"
```

Comment:
Since you use relative paths for the other `ExecStart` lines, why use a specific path here for `env`?

Reply:
`env` isn't necessary, at all... I'll give this another pass later today, as I realize there's no reason to touch the provider-specific configs such as this one